FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Pavan Powani on 03/08/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImagesMapper {
+	struct ImagesRoot: Decodable {
```

Comment:
`ImagesRoot` is an implementation detail of `FeedImagesMapper`, so it should be marked as `private`.
```suggestion
	private struct ImagesRoot: Decodable {
```